src/api/app/models/configuration.rb

```diff
@@ -75,6 +75,10 @@ def ldap_enabled?
     CONFIG['ldap_mode'] == :on
   end
 
+  def ampq_namespace
+    CONFIG['ampq_namespace'] || 'opensuse.obs'
```

Comment:
Is AMQP, not ampq. So, replace it in every event that uses this namespace method too